### ISSUE TITLE
UNMET PEER DEPENDENCY eslint@>=0.8.0

### ISSUE THREAD

**Issue by hellboy81** [nfroidure/eslint-plugin-mongodb#5]
``` sh
 sudo npm i -g eslint
/usr/local/bin/eslint -> /usr/local/lib/node_modules/eslint/bin/eslint.js
/usr/local/lib
└── eslint@1.5.1

sudo npm install --g eslint-plugin-mongodb
/usr/local/lib
├── UNMET PEER DEPENDENCY eslint@>=0.8.0
└── eslint-plugin-mongodb@0.2.2

npm WARN EPEERINVALID eslint-plugin-mongodb@0.2.2 requires a peer of eslint@>=0.8.0 but none was installed.
```

**Comment by nfroidure** [comment]
That's a problem indeed, let just remove it since it doesn't appear in every eslint plugins i just checked.

**Comment by nfroidure** [comment]
@hellboy81 published (0.2.3).

Btw, i you give this plugin a try, can you please give me the settings you ended with here https://github.com/nfroidure/eslint-plugin-mongodb/issues/6 ?
